src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -143,90 +141,79 @@ class TezosNodeOperator(node: TezosRPCInterface) extends LazyLogging {
     ApiOperations.fetchMaxLevel().flatMap{ maxLevel =>
       if(maxLevel == -1) logger.warn("There were apparently no blocks in the database. Downloading the whole chain..")
       getBlockHead(network).flatMap { blockHead =>
-        val headLevel =  blockHead.metadata.header.level
-        val headHash  =  blockHead.metadata.hash
+        val headLevel = blockHead.metadata.header.level
+        val headHash  = blockHead.metadata.hash
         if(headLevel <= maxLevel)
           Try(List[Block]())
         else
-          getBlocks(network, maxLevel+1, headLevel, Some(headHash), followFork)
+          getBlocks(network, maxLevel+1, headLevel, headHash, followFork)
       }
     }
 
   /**
     * Gets the latest blocks from the database using an offset.
     * @param network        Which Tezos network to go against
     * @param offset         How many previous blocks to get from the start
-    * @param startBlockHash The block from which to offset, using the hash provided or the head if none is provided.
+    * @param hash           The block from which to offset, using the hash provided or the head if none is provided.
```

Comment:
would it be better to update the param description with no mention of it being optional?

Reply:
We're going to turn it back to being optional, but good catch!